packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -195,9 +195,13 @@ export const DetailsTile = ({
   hideShare,
   isPlaying,
   isPlayable = true,
+  isPlaylist = false,
+  isPublished = true,
```

Comment:
do we want to provide default args for published, or should that always come from the metadata?

Reply:
I think it will always come from the metadata, but added this bc i wanted the default behavior to ignore the code that was added.